cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -792,6 +792,74 @@ def process_dfe(self, ids: List[str]) -> float:
         """
         return self._client.process_dfe(ids)
 
+    def submit_aces(
+        self,
+        target: str,
+        qubits: List[int],
+        shots: int,
+        num_circuits: int,
+        mirror_depth: int,
+        extra_depth: int,
+        method: Optional[str] = None,
+        noise: Optional[Union[str, cirq.NoiseModel]] = None,
+        error_prob: Optional[Union[float, Tuple[float, float, float]]] = None,
+        tag: Optional[str] = None,
+        lifespan: Optional[int] = None,
+    ) -> str:
+        """Performs a POST request on the `/aces` endpoint.
+
+        Args:
+            target: The device target to characterize.
+            qubits: A list with the qubit indices to characterize.
+            shots: How many shots to use per circuit submitted.
+            num_circuits: How many random circuits to use in the protocol.
+            mirror_depth: The half-depth of the mirror portion of the random circuits.
+            extra_depth: The depth of the fully random portion of the random circuits.
+            method: Which type of method to execute the circuits with.
+            noise: Noise model to simulate the protocol with. It can be either a string or a
+                `cirq.NoiseModel`.
+
+                Valid strings are "symmetric_depolarize", "phase_flip", "bit_flip" and
+                "asymmetric_depolarize".
```

Comment:
```suggestion
            noise: Noise model to simulate the protocol with. It can be either a string or a
                `cirq.NoiseModel`. Valid strings are "symmetric_depolarize", "phase_flip",
                "bit_flip" and "asymmetric_depolarize".
```